src/cmd/release.go

```diff
@@ -54,6 +55,6 @@ func init() {
 	rootCmd.AddCommand(releaseCmd)
 	releaseCmd.AddCommand(gitlabCmd)
 	releaseCmd.AddCommand(githubCmd)
-	gitlabCmd.Flags().StringVarP(&tokenVarName, "token-var-name", "t", "GITLAB_RELEASE_TOKEN", "Environment variable name for GitLab token")
-	githubCmd.Flags().StringVarP(&tokenVarName, "token-var-name", "t", "GITHUB_TOKEN", "Environment variable name for GitHub token")
+	gitlabCmd.Flags().StringVarP(&gitlabTokenVarName, "token-var-name", "t", "GITLAB_RELEASE_TOKEN", "Environment variable name for GitLab token")
```

Comment:
Question: Was this fixing an existing but and is there a new test for the scenario that was exposing the bug? (It may be covered I'm just not sure)

Reply:
Yeah this bug was introduced when @Racer159 helped me refactor the release command. As of now there's no test to cover it due to the complexities of mocking everything required when calling the release command